src/renderers/webgl/WebGLProgram.js

```diff
@@ -282,8 +282,21 @@ THREE.WebGLProgram = ( function () {
 
 		if ( material instanceof THREE.RawShaderMaterial ) {
 
-			prefixVertex = '';
-			prefixFragment = '';
+			prefixVertex = [
+
+				'#define SHADER_NAME ' + material.__webglShader.name,
```

Comment:
I think being consistent with the coding pattern used for the other materials has benefits, but it is up to you.